docs/level-10-moban-dependency-as-git-repo/.moban.yml

```diff
@@ -2,10 +2,10 @@ requires:
   - https://github.com/moremoban/pypi-mobans
 configuration:
   template_dir:
-    - "pypi-mobans:templates"
+    - "repo://pypi-mobans/templates"
```

Comment:
Yep, I was reading more about python file system 2. usercache is equivalent for appdir user cache.

Reply:
however, another thought from me is: usercache is our design choice, meaning user would not have to know 'usercache' and simply remember 'repo://'.

Reply:
but `repo://` doesnt work unless moban has also fetched it before hand.  `mobanrepo://` could be an alias for `usercache://moban/repos/`

Reply:
I think my argument is moban that chooses to checkout in usercache folder, so usercache is moban’s reponsibilty. 

If it is user’s choice, they shall remember it.

Reply:
They still need to learn that `repo://` is needed for any `requires:` entry of type git.  If it is `usercache://moban/repos/` they learn that instead, but then they also learn that it works because moban fetches the repo into `usercache://moban/repos/` so they can go find the cached repo if they want to.

I will accept `repo:` if that is your choice - it is unlikely that anyone would ever build a real fs for protocol `repo` as that isnt a protocol.

`requires://` is another possibility.

Reply:
yes, you are right. both "repo" and "usercache" are new to the user, the first one is really arbitrary, the latter has half of the truth. anyway, user needs to learn a term. so I would accept your argument.

Reply:
Also https://source.android.com/setup/develop/repo and pypi package 'repofs' ;)

Reply:
@jayvdb , usercache requires 'user' parameter, which will break team collaboration unless

a) we use 'repo'
b) we figure out how to get 'user'..

Reply:
Even though we can get 'user', our current data loader does not support variable in data. It is a bit far away target.